program_install_for_soma.sh

```diff
@@ -6,18 +6,20 @@ REDIS_VERSION='3.2.8'
 
 if [ ! -d "redis" ]; then
     echo "   3) INSTALLING REDIS-$REDIS_VERSION"
-    wget http://download.redis.io/releases/redis-$REDIS_VERSION.tar.gz
+    wget http://download.redis.io/releases/redis-$REDIS_VERSION.tar.gz -P $REPO_ROOT_PATH
     tar zxvf redis-$REDIS_VERSION.tar.gz
     cd redis-$REDIS_VERSION && make
     mv $REPO_ROOT_PATH/redis-$REDIS_VERSION $REPO_ROOT_PATH/redis
+    cd ..
```

Comment:
how about changing this "cd .." to "cd $REPO_ROOT_PATH"
I think it is more robust